tests/test_asyncio.py

```diff
@@ -0,0 +1,24 @@
+import asyncio
```

Comment:
It seems strange to have tests testing `asyncio` here?

Reply:
Woops!! Yeah I still don't know why the one xfailed test isn't raising the `TimeoutError` like I believe it should. I was trying to reproduce that behavior in a minimal example, accidentally added and commited it.